test/v2/services/SubmitForeignPropertyBsasServiceSpec.scala

```diff
@@ -34,7 +34,7 @@ class SubmitForeignPropertyBsasServiceSpec extends UnitSpec {
 
   private val nino = Nino("AA123456A")
   private val id = "f2fb30e5-4ab6-4a29-b3c1-c7264259ff1c"
-  private val correlationId = "X-123"
+  private implicit val correlationId = "X-123"
```

Comment:
Please remove once you have added the **implicit val correlationId: String = "X-123"** to the ServiceSpec and this class extends it instead (Change from UnitSpec to ServiceSpec)